infra/main.bicep

```diff
@@ -229,6 +230,7 @@ module backend 'core/host/appservice.bicep' = {
       APPLICATIONINSIGHTS_CONNECTION_STRING: useApplicationInsights ? monitoring.outputs.applicationInsightsConnectionString : ''
       // Shared by all OpenAI deployments
       OPENAI_HOST: openAiHost
+      OPENAI_BASE_URL: openAiBaseUrl
```

Comment:
I think this should be AZURE_OPENAI_BASE_URL or AZURE_OPENAI_CUSTOM_URL, to clarify that it's being used with an Azure client.